daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -144,6 +144,7 @@ final case class Compiler(packages: PackageId PartialFunction Package) {
           case BFoldl => SEBuiltinRecursiveDefinition.FoldL
           case BFoldr => SEBuiltinRecursiveDefinition.FoldR
           case BEqualList => SEBuiltinRecursiveDefinition.EqualList
+          case BCoerceContractId => SEAbs(1, SEVar(1))
```

Comment:
@neil-da That's the Scala implementation: `\x -> x`.

Reply:
So simples :)